src/components/Books/Main/Book.tsx

```
@@ -0,0 +1,86 @@
+          <span className={`${styles.main_booksContainer_book_price}`}>
+            {book.price}$
+          </span>
+          <div className={`${styles.main_booksContainer_book_stars}`}>
+            {stars.map((star) => {
+              if (star)
+                return (
+                  <div
+                    className={`${styles.main_booksContainer_book_stars_star_active}`}
+                  >
+                    <FaStar />
+                  </div>
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)